app/controllers/chatrooms_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ChatroomsController < ApplicationController
+  
+  def new
+    @chatroom = Chatroom.new
+  end
+
+  def create
+    @chatroom = Chatroom.new chatroom_params
+    if @chatroom.save
+      flash[:info] = t ".controller.chatrooms.create_chatroom_succ"
+      redirect_to new_chatroom_path
```

Comment:
sau khi tạo xong lại về trang tạo tiếp thì có vẻ k hay lắm nhỉ?
em có thể cho 2 nút: 1 nút "create" và 1 nút "create and continue" thì sẽ hợp lý hơn

Reply:
sau khi tạo xong thì sẽ chuyến sang trang bên trong chatroom. Do chức năng đấy chưa hoàn thiện nên tạm thời bọn e để "redirect_to" new_chatroom_path ạ :D